packages/dbgpt-client/src/dbgpt_client/flow.py

```
@@ -40,14 +40,20 @@ async def update_flow(client: Client, flow: FlowPanel) -> FlowPanel:
+            raise ClientException(
+                reason="Failed to update flow: flow.uid is required"
+            )
+        res = await client.put(f"/awel/flows/{flow.uid}", flow.to_dict())
         result: Result = res.json()
         if result["success"]:
             return FlowPanel(**result["data"])
         else:
             raise ClientException(status=result["err_code"], reason=result)
+    except ClientException:
+        raise
     except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)